### ISSUE TITLE
trying to flash TX16s with 3.0RC2 and it says "bad size given" while trying to update over WIFI.

### ISSUE THREAD

**Issue by gminton86** [ExpressLRS/ExpressLRS#1805]
I tried the 3.0 RC1 and RC2 and they both say the same thing 2.5.1 works fine.

**Comment by pkendall64** [comment]
This seems to be a common issue and users who have this problem will have to perform an updated using USB.

**Comment by pkendall64** [comment]
We've figured out the issue and provided a solution in the pinned comments in the #help-and-supprt channel on discord.

**Comment by gminton86** [comment]
Thanks I got it to work with your advice.

**Comment by jurgelenas** [comment]
> We've figured out the issue and provided a solution in the pinned comments in the #help-and-supprt channel on discord.

is it documented on expresslrs.org?

**Comment by pkendall64** [comment]
> > We've figured out the issue and provided a solution in the pinned comments in the #help-and-supprt channel on discord.
> 
> is it documented on expresslrs.org?

No, not yet. Waiting to here back from deadbyte and others on their testing efforts.
